napari/_qt/qt_main_window.py

```diff
@@ -376,11 +393,21 @@ def _add_window_menu(self):
         exit_action.setShortcut("Ctrl+W")
         exit_action.setStatusTip('Close napari window')
         exit_action.triggered.connect(self._qt_window.close)
+
+        clear_action = QAction("Clear Menu", self._qt_window)
```

Comment:
I like the behavior of having some menu item that clears all plugin dock widgets.  but I think the word "clear menu" is a little confusing, because it made me think I was just removing some items from the _menu_ (not that it would be closing widgets that I had been working on in the viewer).

Reply:
`Remove Dock Widgets` or `Clear Dock Widgets`?

Reply:
either `Remove` or `Close All Dock Widgets`?

Reply:
My top choice is probable `Remove Dock Widgets`. @jni?

Reply:
renamed `Remove Dock Widgets`